src/content/api/assets.mdx

```diff
@@ -0,0 +1,551 @@
+      -d '{
+        "name": "Centrapay Cafe",
+        "type": "org"
+      }'
+    ```
+
+    ```json
+    {
+      "items": [
+        {
+          "ref": "adRGJqMyMhmGfnDXasRZ",
+          "type": "transfer",
+          "kind": "refund",
+          "refType": "payment-request",
+          "assetId": "WRhAxxWpTKb5U7pXyxQjjY",
+          "srcParty": "Coffee Ltd",
+          "destParty": "crn::user:5a3b1ba7-d01k-409f-ld0a-jd81k0ald",
+          "createdAt": "2022-03-31T20:36:08.562Z",
+          "createdBy": "crn::service:payments-api",
+          "srcAssetId": "Jd9a89ZESjjCuUD9DJD9Al",
+          "destAssetId": "WRhAxxWpTKb5U7pXyxQjjY",
+          "amount": "2000",
+          "activityType": "value-in",
+          "activityNumber": "3"
+        },
+        {
+          "ref": "adRGJqMyMhmGfnDXasRZ",
+          "type": "transfer",
+          "kind": "payment",
+          "refType": "payment-request",
+          "assetId": "WRhAxxWpTKb5U7pXyxQjjY",
+          "srcParty": "crn::user:5a3b1ba7-d01k-409f-ld0a-jd81k0ald",
+          "destParty": "Coffee Ltd",
+          "createdAt": "2022-03-31T20:35:54.717Z",
+          "createdBy": "crn::service:payments-api",
+          "srcAssetId": "WRhAxxWpTKb5U7pXyxQjjY",
+          "destAssetId": "Jd9a89ZESjjCuUD9DJD9Al",
+          "amount": "3000",
+          "activityType": "value-out",
+          "activityNumber": "2"
+        },
+        {
+          "ref": "H4SZKwMcU9VCmnGEqDA7Mn",
+          "type": "increment-balance",
+          "kind": "topup",
+          "refType": "topup",
+          "assetId": "WRhAxxWpTKb5U7pXyxQjjY",
+          "createdAt": "2022-03-31T02:37:47.207Z",
+          "createdBy": "crn::application:rhea",
+          "destAssetId": "WRhAxxWpTKb5U7pXyxQjjY",
+          "amount": "7600",
+          "activityType": "value-in",
+          "activityNumber": "1"
+        }
+      ]
+    }
+  ```
+  </CodeGroup>
+</Endpoint>
+
+---
+
+<Endpoint
+  method="POST"
+  path="/api/assets/{assetId}/archive"
+>
+  ### Archive Asset
+
+  Archive supported asset types by asset id. Currently only gift cards may be archived.
+
+  ### Attributes
+  No Attributes.
+
+  <CodeGroup slot="code-examples">
+    ```bash
+    curl -X POST https://service.centrapay.com/api/assets/L75M3L56N2PtBSt8g7uXLU/archive \
+      -H "X-Api-Key: $api_key"
+    ```
+
+    ```json
+    {
+      "id": "L75M3L56N2PtBSt8g7uXLU",
+      "accountId": "Te2uDM7xhDLWGVJU3nzwnh",
+      "category": "giftcard",
+      "type": "epay.nzd.main",
+      "issuer": "ezipay",
+      "externalId": "23403283262",
+      "description": "$60 Acme Giftcard",
+      "initialBalance": "6000",
+      "balance": "0",
+      "balanceUpdatedAt": "2021-01-01T00:00:00.000Z",
+      "expiresAt": "2020-12-31T00:00:00.000Z",
+      "createdAt": "2020-05-01T12:30:00.000Z",
+      "status": "archived"
+    }
+    ```
+
+  </CodeGroup>
+
+  ### Errors
+  | Status | Code                    | Description                                         |
```

Comment:
@alexBinns I've just realised that we should probably be using the Properties component to document errors. https://www.figma.com/file/IgRlQWJTyGoZhlF4fVKD6B/Centrapay-Docs-(Tailwind)?type=design&node-id=1651-4070&mode=design&t=KlKVfCHVGJO9uraz-0

I've been using tables as well 😅 What do you think?

Reply:
<img width="1128" alt="image" src="https://github.com/centrapay/centrapay-docs/assets/70119888/ecfc8180-5859-401d-ab62-d76ec8841f48">

Reply:
I think tables is ok for now - this is what is in the designs for errors. We just need to format it 👀

Reply:
Sounds good - is there a link for this design? The link above shows them looking identical to the properties section

Reply:
@AndyClifford sorry I completely missed that link and hadn't seen the updated design 😅 
Let's update to match the design 👯

Reply:
Updated now!